src/software/thunderscope/robot_diagnostics/drive_and_dribbler_widget.py

```diff
@@ -54,6 +88,33 @@ def value_change(self, value: float) -> str:
         value_str = "%.2f" % value
         return value_str
 
+    def setup_drive_switch_radio(self, title: str) -> QGroupBox:
```

Comment:
Right now neither radio button is checked when Thunderscope first starts up, so you should check one of them by default